pkg/config/config.go

```diff
@@ -85,7 +85,8 @@ func (el *EnvList) Add(key, value string) {
 func (el EnvList) Validate() error {
 	const op errors.Op = "EnvList.Validate"
 	for _, env := range el {
-		if strings.Count(env, "=") != 1 {
+		// excluding GODEBUG because it can have multiple "=", such as GODEBUG=netdns=cgo
+		if strings.Count(env, "=") != 1 && !strings.HasPrefix(env, "GODEBUG=") {
```

Comment:
maybe I was too strict to require the count to be 1, maybe we should just say `if strings.Count(env, "=") < 1` and therefore we don't need to make a special case for GODEBUG. This way, we can allow other env vars that might need multiple equal signs.

Reply:
Makes sense to me! Updated